### PR TITLE
*: refine TiDB Operator docs

### PR DESCRIPTION
**Pull request by lilin90**
@tennix PTAL

### CODE REVIEW

**Comment by tennix** [comment]
@gregwebs The default static route in DinD would be lost if the node (outer docker container) restarts causing pods failure. With flannel, the routes info are persisted in etcd.

File: docs/local-dind-tutorial.md [hunk 1]
```diff
@@ -166,40 +182,50 @@ To access the TiDB cluster, use `kubectl port-forward` to expose the services to
...
 ```sh
 $ helm delete tidb-cluster --purge
 ```
 
-> **Note:** This only deletes running pods and other resources, the data is persisted. If you don't need the data anymore, run the following commands to cleanup the data. (Be careful, this permanently delete the data).
+> **Note:** This only deletes the running pods and other resources, the data is persisted. If you do not need the data anymore, run the following commands to clean up the data. (Be careful, this permanently deletes the data).
 
 ```sh
-$ kubectl get pv -l cluster.pingcap.com/namespace=tidb -o name | xargs -i kubectl patch {} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
+$ kubectl get pv -l cluster.pingcap.com/namespace=tidb -o name | xargs -I {} kubectl patch {} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
```
[the remark this answers is not in the document]
Comment (gregwebs) [reply docs/local-dind-tutorial.md]: Tried with the latest, I still don't see any labels on my PV.

[the remark this answers is not in the document]
Comment (weekface) [reply docs/local-dind-tutorial.md]: This label is added after it bound with a PVC: https://github.com/pingcap/tidb-operator/blob/master/pkg/controller/pv_control.go#L115

[the remark this answers is not in the document]
Comment (gregwebs) [reply docs/local-dind-tutorial.md]: It is working now on the latest. `describe pv` still doesn't show the label, but the label query works.

**Comment by gregwebs** [comment]
In our published tutorial, we used the text: "This local deployment is solely meant to give you a taste of TiDB Operator for testing and experimentation, not for production use. It’s still undergoing testing from our team and the open-source community, in which we encourage you to participate."

Probably even better is to have production-readiness be part of the roadmap.

**Comment by tennix** [comment]
@gregwebs We can add that in later PRs.

**Comment by gregwebs** [comment]
CNI is failing (PD won't connect) on Linux for me. I would suggest

``` bash
CNI_PLUGIN=$((uname -s | grep -i darwin 1>/dev/null) && echo "flannel")
```

**Comment by tennix** [comment]
I've tested flannel on my ArchLinux, it works pretty well.
image
The failure for pd and tidb pod are because the Docker images took too long to pull not because flannel network plugin.

**Comment by gregwebs** [comment]
Looks like your Linux setup is different than mine (Ubuntu). With flannel, I get connection failure in the pd error logs.

```
2018/08/14 18:07:15.766 main.go:92: [fatal] run server failed: --initial-cluster must include demo-pd-0=http://demo-pd-0.demo-pd-peer.tidb.svc:2380 given --initial-advertise-peer-urls=http://demo-pd-0.demo-pd-peer.tidb.svc:2380
```

**Pull request merged**